hudi-examples/hudi-examples-spark/pom.xml

```diff
@@ -190,6 +190,12 @@
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
         </dependency>
 
+        <!-- Hadoop -->
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-auth</artifactId>
+        </dependency>
+
```

Comment:
SparkQuickStartTest would throw NoSuchMethodError without this

Reply:
good find. so can we now re-enable spark 3.2 quickstart test in GH action? check out bot.yml

Reply:
Highly likely. maybe we need a seperate jira to track that?

Reply:
Filed this one: https://issues.apache.org/jira/browse/HUDI-4479